scraper/spiders/__init__.py

```diff
@@ -55,7 +55,7 @@ class SmtStories(CrawlSpider):
     # write the URLS for stories to scrape to a local file
     write_nodes_to_file(db_nodes, 'articles')
 
-    allowed_domains = ["socialmediatoday.com"]
+    allowed_domains = [site_url]
```

Comment:
@sw3dish Does this still work even though we're now giving it a full URL instead of only a domain?